infra/aws/terraform/kops-infra-ci/eks.tf

```diff
@@ -119,6 +119,21 @@ module "eks" {
         }
       }
 
+      # NVMe instance store volumes are automatically enumerated and assigned a device
+      pre_bootstrap_user_data = <<-EOT
+        cat <<-EOF > /etc/profile.d/bootstrap.sh
```

Comment:
is there an extra `-` before EOF?

Reply:
never mind .. found it in other places like https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/d2c87b184d2115bfaf6693e66fa2e0b8b470f4ef/patterns/stateful/main.tf#L176